src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenCheck.java

```diff
@@ -35,22 +35,26 @@
  * Rationale: Certain language features can harm readability, lead to
  * confusion or are not obvious to novice developers. Other features
  * may be discouraged in certain frameworks, such as not having
- * native methods in EJB components.
+ * native methods in Enterprise JavaBeans components.
  * </p>
+ * <ul>
+ * <li>
+ * Property {@code tokens} - tokens to check
+ * Default value is:
+ * <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#LABELED_STAT">
+ * LABELED_STAT</a>.
+ * </li>
+ * </ul>
  * <p>
- * An example of how to configure the check is:
+ * To configure the check to find token LITERAL_NATIVE:
  * </p>
  * <pre>
- * &lt;module name="IllegalToken"/&gt;
- * </pre>
```

Comment:
Unless you have been removing them, I think we should keep the default module configurations.

Reply:
done.